test_team_guardiansofpii.py

```diff
@@ -12,6 +12,22 @@ def test_anonymize_text(self):
 
     def test_email_address(self):
         """Test email_address functionality"""
+     test_string = "My email address is example@example.com"
```

Comment:
add #Positive Testcase above the first test string

Reply:
Are you using a codespace?  Indents should have been mostly automatic